src/sql/base/browser/ui/table/media/slickColorTheme.css

```diff
@@ -17,18 +17,24 @@
 }
 
 .slick-cell a,
-.slick-cell a:link,
```

Comment:
removed the style for `:link` since it only makes sense when we define `:visited` to differentiate them.